tests/optimizer/test-function-eliminator-double-parsed-correctly-output.js

```diff
@@ -0,0 +1,14 @@
+// EMSCRIPTEN_START_ASM
+var asm = (function(global, env, buffer) {
+ "use asm";
+ var e = 0;
+ 
+// EMSCRIPTEN_START_FUNCS
+function a() {
+ var c = 0.0;
```

Comment:
Yup, I wanted to explicitly make sure that we end up with a 0.0 (instead of 0 which was the bug before).